### ISSUE TITLE
hyperv: Unable to start VM: start: exit status 1

### ISSUE THREAD

**Issue by rossbrigoli** [kubernetes/minikube#5226]
**The exact command to reproduce the issue**: 
minikube start --vm-driver=hyperv --hyperv-virtual-switch=External-VM-Switch

**The full output of the command that failed**:<details>
`* minikube v1.3.1 on Microsoft Windows Server 2016 Standard 10.0.14393 Build 14393
`* Tip: Use 'minikube start -p <name>' to create a new cluster, or 'minikube delete' to delete this one.
`* Starting existing hyperv VM for "minikube" ...
E0829 12:29:45.457415   13628 start.go:723] StartHost: start: exit status 1
`*
`X Unable to start VM: start: exit status 1
`*
`* Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
`  - https://github.com/kubernetes/minikube/issues/new/choose

</details>

**The output of the `minikube logs` command**:<details>
Log folder is emtpy
</details>

**The operating system version**: Windows Server 2016 Standard 10.0.14393 Build 14393

**Comment by tstromberg** [comment]
Do you mind sharing the output of:

```
minikube start --vm-driver=hyperv --hyperv-virtual-switch=External-VM-Switch --alsologtostderr -v=8
```

and:

```
Get-VMSwitch
```

If the switch name is correct, it may be possible to get around this issue using `minikube delete`. Please let me know what you find!

**Comment by tstromberg** [comment]
@rossbrigoli : I believe this issue is now addressed by minikube v1.4, as it no longer requires an external virtual switch to be configured. If you still see this issue with minikube v1.4 or higher, please reopen this issue by commenting with `/reopen`
